hooks/useCollections.ts

```
@@ -50,15 +62,15 @@ export function useCollections() {
 
   useEffect(() => {
     if (data?.pages.length) {
-      setCollections(
-        data?.pages?.reduce((acc: Collection[], page) => {
+      setCollections([
+        ...(data?.pages?.reduce((acc: Collection[], page) => {
           return [...acc, ...page.collections];
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)